docs/gateway-operator-api-reference.md

```diff
@@ -1299,6 +1300,8 @@ WatchNamespaces defines the namespaces to watch for resources
 
 _Appears in:_
 - [ControlPlaneOptions](#controlplaneoptions)
+- [ControlPlaneOptions](#controlplaneoptions)
```

Comment:
we should use an anchor for #controlplaneoptions that belongs to the proper CRD API version

Reply:
Please see my comment in https://github.com/Kong/kubernetes-configuration/pull/441#discussion_r2126097550